### PR TITLE
Data Preview module

### LINKED ISSUE #38: Data Previewer Module
As an app user
In order to start my analysis
I want to see the loaded datasets.

Implementation guide:
- Create a module (both server and ui part) - modify `shiny/m_preview.R` script.
- Use `sidebarLayout` as in home module.
- Use `DT` package to render out the table.
- In the encodings panel please create a select input to choose the dataset

BONUS:
- Add a checkbox to see un-filtered data.

### PR DESCRIPTION
**Pull request by pawelru**
close #38

### CODE REVIEW

File: programs/shiny/m_preview.R [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
   ns <- NS(id)
-  "Data Preview"
+  DT::dataTableOutput(ns("table"))
 }
-m_preview_srv <- function(id, list_reactive_datasets) {
+m_preview_single_srv <- function(id, dataset) {
   moduleServer(
     id,
     function(input, output, session) {
+      output$table <- DT::renderDataTable(dataset(), options=list(scrollX=T))
```
Comment (pawelru) [inline programs/shiny/m_preview.R:9]: how about adding additional option to `DT::renderDataTable`, e.g. I propose blah blah

Comment (simsj1) [reply programs/shiny/m_preview.R:9]: fantastic idea

**Review by simsj1 — APPROVED** [review]
Workshop presenter's code gets approved automatically :-)

File: programs/shiny/m_preview.R (hunk 1 above)
Comment (LucaFoppoli) [reply programs/shiny/m_preview.R:9]: Same here! Cool tool this is (spoken like Yoda ops)

File: programs/shiny/m_preview.R [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
+    }
+  )
+}
 
+m_preview_ui <- function(id) {
+  ns <- NS(id)
+  tabsetPanel(
+    tabPanel("ADSL", m_preview_single_ui(ns("ADSL"))),
+    tabPanel("ADAE", m_preview_single_ui(ns("ADAE"))),
+    tabPanel("ADVS", m_preview_single_ui(ns("ADVS")))
```
Comment (LucaFoppoli) [inline programs/shiny/m_preview.R:19]: Trying to leave a comment

**Review by LucaFoppoli — APPROVED** [review]
So everything goes back to the person who first wrote the code?

File: programs/shiny/m_preview.R (hunk 2 above)
Comment (pawelru) [reply programs/shiny/m_preview.R:19]: Thank you for your comment!

**Pull request merged**